src/mock_display.rs

```diff
@@ -373,6 +381,36 @@ where
 
         target
     }
+
+    /// Compares the display to another display.
+    ///
+    /// The following color code is used to show the difference between the displays:
+    ///
+    /// | Color               | Description                                                   |
+    /// |---------------------|---------------------------------------------------------------|
+    /// | None                | The color of the pixel is equal in both displays.             |
+    /// | Some(Rgb888::GREEN) | The pixel was only set in `self`                              |
+    /// | Some(Rgb888::RED)   | The pixel was only set in `other`                             |
+    /// | Some(Rgb888::BLUE)  | The pixel was set to a different colors in `self` and `other` |
```

Comment:
Do you think it would add too much noise to print these rules to the console with the diff table? Not as verbose as the table presented here, but a quick reminder might be useful. A (hopefully) single line sentence would be ideal.

Reply:
That's a good idea, but I would like to do this as part of a future PR. The `mock_display.rs` file is getting long and I would like to move the code into a `mock_display` to clean it up a bit.